src/server/stream_family.cc

```diff
@@ -952,12 +952,16 @@ struct ClaimOpts {
   int64 delivery_time = -1;
   int retry = -1;
   uint8_t flags = 0;
+  int32_t count = 100;  // only for XAUTOCLAIM
+  streamID start;       // only for XAUTOCLAIM
```

Comment:
Here and  for `end_id` below - consider initialize to a default value

Reply:
fixed